src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```
@@ -1871,6 +1872,37 @@ because xbuild doesn't support framework reference assemblies.
 <Target Name="_LinkAssemblies"
   DependsOnTargets="_ResolveAssemblies;_CreatePackageWorkspace;$(_BeforeLinkAssemblies);_GenerateJniMarshalMethods;_LinkAssembliesNoShrink;_LinkAssembliesShrink" />
 
+<Target Name="_PrepareNativeAssemblySources">
+  <PrepareAbiItems
+    BuildTargetAbis="@(_BuildTargetAbis)"
+    ItemNamePattern="$(_NativeAssemblySourceDir)typemaps.@abi@.s">
+      <Output TaskParameter="OutputItems" ItemName="_TypeMapAssemblySource" />
+  </PrepareAbiItems>
+
+</Target>
+<Target Name="_GenerateTypeMaps"
+    DependsOnTargets="_PrepareNativeAssemblySources"
+    Inputs="@(ResolvedAssemblies);@(ResolvedUserAssemblies)"
+    Outputs="$(_AndroidStampDirectory)_GenerateTypeMaps.stamp">
+  <GenerateTypeMaps
+    ResolvedAssemblies="@(_ResolvedAssemblies)"
+    ResolvedUserAssemblies="@(_ResolvedUserAssemblies)"
```

Can this one be?
```suggestion
    ResolvedUserAssemblies="@(_ResolvedUserMonoAndroidAssemblies)"
```
Then the task can skip looking at netstandard or pcl assemblies.